pyppeteer/page.py

```diff
@@ -1105,11 +1080,9 @@ def predicate(response: Response) -> bool:
 
         * ``userAgent`` (str): user agent string.
         """
-        options = merge_dict(options, kwargs)
-        # TODO: if options does not have viewport or userAgent,
         # skip its setting.
```

Comment:
No need to, otherwise they could just use the setViewport/setUserAgent themselves
```suggestion
```